src/or/dirauth/shared_random_state.c

```diff
@@ -9,11 +9,13 @@
  **/
 
 #define SHARED_RANDOM_STATE_PRIVATE
+#define TOR_SHARED_RANDOM_COMMON_PRIVATE
```

Comment:
One C file should never define another C file's "private" macro.  Is there some way we can avoid doing this?  If something is private to shared_random_common, literally nothing else besides the tests should be looking at it.

Reply:
Yes actually... I believe this is an artifact and it builds perfectly without it!

Fixup commit e614ae1a03